tests/packer/km-az-test.pkr.hcl

```diff
@@ -119,9 +124,9 @@ build {
   }
 
   provisioner "shell" {
-    script          = "packer/scripts/km-test.sh"
+    script = "packer/scripts/km-test.sh"
     // double sg invocation to get docker into the process's grouplist but not the primary group.
-    execute_command = "chmod +x {{ .Path }}; {{ .Vars }} sg docker -c 'sg ubuntu {{ .Path }}'"
+    execute_command = "chmod +x {{ .Path }}; {{ .Vars }} sg docker -c 'sg ${var.ssh_user} {{ .Path }}'"
```

Comment:
Just a note here.
The argument to sg is a groupname and var.ssh_user is a username.
This works because the username is the same as the user's primary groupname.
I guess this is some sort of convention in linux these days.
We could probably do something that is the equivalent of:

sg `id -n -g` xxxxxx

We would need to fiddle around with where we run id -n -g to ensure we didn't get "docker".

I'm not asking you to do anything here.  Just commenting about it.